.pre-commit-config.yaml

```diff
@@ -0,0 +1,48 @@
+-   repo: https://github.com/pre-commit/pre-commit-hooks.git
+    sha: v0.9.1
+    hooks:
+    -   id: trailing-whitespace
+        language_version: python2.7
+    -   id: end-of-file-fixer
+        language_version: python2.7
+    -   id: autopep8-wrapper
+        language_version: python2.7
+    -   id: check-docstring-first
+        language_version: python2.7
+    -   id: check-json
+        language_version: python2.7
+    -   id: check-yaml
+        files: ^(?!.*template).*\.(yaml|yml|eyaml)$
+        language_version: python2.7
+    -   id: requirements-txt-fixer
+        language_version: python2.7
+    -   id: flake8
+        exclude: ^docs/source/conf.py$
+        language_version: python2.7
+    -   id: check-ast
+        language_version: python2.7
+-   repo: https://github.com/asottile/reorder_python_imports.git
+    sha: v0.3.5
+    hooks:
+    -   id: reorder-python-imports
+        language_version: python2.7
```

Comment:
you _should_ be able to drop the `language_version` stuff everywhere as long as you're using `pre-commit>0.15.1`: https://github.com/pre-commit/pre-commit/blob/master/CHANGELOG.md#0151

Reply:
`pre-commit --version` is 0.18.1, version guessed is 3.6 which fails flake8 and check-ast. how can i make it guess 2.7?

Reply:
0.18.1 is pretty old too.  It guesses based on the currently running interpreter, otherwise yeah  you should manually specify this.

If you're running `pre-commit` through `tox`, you can control the interpreter by settings `basepython = ...`